src/sentry/migrations/0470_convert_unresolved_to_set_escalating_activitytype.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 2.2.28 on 2023-05-26 07:32
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.types.activity import ActivityType
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def convert_to_set_escalating(apps, schema_editor):
+    Activity = apps.get_model("sentry", "Activity")
+
+    for activity in RangeQuerySetWrapperWithProgressBar(
+        Activity.objects.filter(type=ActivityType.SET_UNRESOLVED)
```

Comment:
I think this might end up timing out... there's an index here but we're using `RangeQuerySetWrapperWithProgressBar` which sorts by id, which can break index usage.

It might be worth trying just to see if it works out, but if it doesn't you might just need to iterate the whole table and filter by type in python instead.